### DISCUSSION TITLE
ccf-validation-tools/Fallopian_tube/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#211] [General]
# ccf-validation-tools/Fallopian_tube/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Fallopian_tube/comments/

**Comment by aleixpuigb** [comment]
**New Term in CL**
Rows 104, 108, 112, 116 and 117: Peg cell (intercalary cell) - CL:4033014

**Comment by emquardokus** [comment]
**Fallopian_Tube_v1.2_DRAFT**
https://docs.google.com/spreadsheets/d/16DHu7R9MC2B_fP7uRiDYcDIUvFs6SwHmZWgwpzuJwfI/edit#gid=991519552

Edits 9-13-2023 2:30 am Eastern Ellen
**Different labels**
Fixed In row 44, the term UBERON:0012648 has different name/label in the source ontology. The name/label in the ASCT+B table is __ and the one in the ontology is ampulla of uterine tube. For reference, the given name/label by SMEs is ampulla of Fallopian tube. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

Fixed In row 83, the term UBERON:8410010 has different name/label in the source ontology. The name/label in the ASCT+B table is __ and the one in the ontology is fimbria of uterine tube. For reference, the given name/label by SMEs is fimbria of Fallopian tube. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

Fixed n row 28, the term UBERON:0003889 has different name/label in the source ontology. The name/label in the ASCT+B table is __ and the one in the ontology is fallopian tube. For reference, the given name/label by SMEs is Fallopian tube. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

Fixed In row 28, the term UBERON:0016632 has different name/label in the source ontology. The name/label in the ASCT+B table is __ and the one in the ontology is isthmus of fallopian tube. For reference, the given name/label by SMEs is isthmus of Fallopian tube. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

Fixed In row 76, the term UBERON:0012331 has different name/label in the source ontology. The name/label in the ASCT+B table is __ and the one in the ontology is mesosalpinx. For reference, the given name/label by SMEs is mesosalpinx. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

Fixed In row 60, the term UBERON:0003984 has different name/label in the source ontology. The name/label in the ASCT+B table is __ and the one in the ontology is uterine tube infundibulum. For reference, the given name/label by SMEs is infundibulum of Fallopian tube. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Blank ontology ID**
**ADDED CL ID: fibroblast	CL:0000057**
In row 13, no term id was found for the name/label Fibroblast.
In row 29, no term id was found for the name/label Fibroblast.
In row 45, no term id was found for the name/label Fibroblast.
In row 61, no term id was found for the name/label Fibroblast.
In row 84, no term id was found for the name/label Fibroblast.

**ADDED CL ID: macrophage	CL:0000235**
In row 15, no term id was found for the name/label macrophage.
In row 31, no term id was found for the name/label macrophage.
In row 47, no term id was found for the name/label macrophage.
In row 63, no term id was found for the name/label macrophage.
In row 86, no term id was found for the name/label macrophage.

**ADDED CL ID: CD8-positive, alpha-beta T cell	CL:0000625**
In row 16, no term id was found for the name/label CD8+ T-cells.
In row 32, no term id was found for the name/label CD8+ T-cells.
In row 48, no term id was found for the name/label CD8+ T-cells.
In row 64, no term id was found for the name/label CD8+ T-cells.
In row 87, no term id was found for the name/label CD8+ T-cells.

**ADDED CL ID: natural killer cell	CL:0000623**
In row 17, no term id was found for the name/label Natural killer cell.
In row 33, no term id was found for the name/label Natural killer cell.
In row 49, no term id was found for the name/label Natural killer cell.
In row 65, no term id was found for the name/label Natural killer cell.
In row 88, no term id was found for the name/label Natural killer cell.

**ADDED CL ID: CD4-positive, alpha-beta T cell	CL:0000624**
In row 18, no term id was found for the name/label CD4+ T-cell.
In row 34, no term id was found for the name/label CD4+ T-cell.
In row 50, no term id was found for the name/label CD4+ T-cell.
In row 66, no term id was found for the name/label CD4+ T-cell.
In row 89, no term id was found for the name/label CD4+ T-cell.

**ADDED CL ID: Langerhans cell	CL:0000453**
In row 19, no term id was found for the name/label Langerhans cell.
In row 35, no term id was found for the name/label Langerhans cell.
In row 51, no term id was found for the name/label Langerhans cell.
In row 67, no term id was found for the name/label Langerhans cell.
In row 90, no term id was found for the name/label Langerhans cell.

**ADDED CL ID: monocyte	CL:0000576**
In row 20, no term id was found for the name/label monocyte.
In row 36, no term id was found for the name/label monocyte.
In row 52, no term id was found for the name/label monocyte.
In row 68, no term id was found for the name/label monocyte.
In row 91, no term id was found for the name/label monocyte.

**ADDED CL ID: dendritic cell	CL:0000451**
In row 21, no term id was found for the name/label dendritic cell.
In row 37, no term id was found for the name/label dendritic cell.
In row 53, no term id was found for the name/label dendritic cell.
In row 69, no term id was found for the name/label dendritic cell.
In row 92, no term id was found for the name/label dendritic cell.

**ADDED CL ID: mast cell	CL:0000097**
In row 22, no term id was found for the name/label mast cell.
In row 38, no term id was found for the name/label mast cell.
In row 54, no term id was found for the name/label mast cell.
In row 70, no term id was found for the name/label mast cell.
In row 93, no term id was found for the name/label mast cell.

**ADDED CL ID: B cell	CL:0000236**
In row 23, no term id was found for the name/label B cell.
In row 39, no term id was found for the name/label B cell.
In row 55, no term id was found for the name/label B cell.
In row 71, no term id was found for the name/label B cell.
In row 94, no term id was found for the name/label B cell.

**ADDED CL ID: plasma cell	CL:0000786**
In row 24, no term id was found for the name/label plasma cell.
In row 40, no term id was found for the name/label plasma cell.
In row 56, no term id was found for the name/label plasma cell.
In row 72, no term id was found for the name/label plasma cell.
In row 95, no term id was found for the name/label plasma cell.

**ADDED CL ID: neutrophil	CL:0000775**
In row 25, no term id was found for the name/label neutrophil.
In row 41, no term id was found for the name/label neutrophil.
In row 57, no term id was found for the name/label neutrophil.
In row 73, no term id was found for the name/label neutrophil.
In row 96, no term id was found for the name/label neutrophil.

**ADDED CL ID: endothelial cell	CL:0000115**
In row 27, no term id was found for the name/label Endothelial cell.
In row 43, no term id was found for the name/label Endothelial cell.
In row 59, no term id was found for the name/label Endothelial cell.
In row 75, no term id was found for the name/label Endothelial cell.
In row 98, no term id was found for the name/label Endothelial cell.

**ADDED CL ID: epithelial cell	CL:0000066**
In row 76, no term id was found for the name/label epithelial cell.

**replaced  intercalary cell. with peg cell	CL:4033014**
In row 104, no term id was found for the name/label intercalary cell.
In row 108, no term id was found for the name/label intercalary cell.
In row 112, no term id was found for the name/label intercalary cell.
In row 116, no term id was found for the name/label intercalary cell.
In row 117, no term id was found for the name/label intercalary cell.

**replaced ciliated epithelial cell with fallopian tube ciliated cell	CL:4030007**
In row 121, no term id was found for the name/label ciliated epithelial cell.
In row 123, no term id was found for the name/label ciliated epithelial cell.
In row 125, no term id was found for the name/label ciliated epithelial cell.
In row 127, no term id was found for the name/label ciliated epithelial cell.
In row 129, no term id was found for the name/label ciliated epithelial cell.

**replaced secretory epithelial cell with fallopian tube secretory epithelial cell	CL:4030006**
In row 122, no term id was found for the name/label secretory epithelial cell.
In row 124, no term id was found for the name/label secretory epithelial cell.
In row 128, no term id was found for the name/label secretory epithelial cell.
In row 130, no term id was found for the name/label secretory epithelial cell.

**fixed all DOIs**
DOI: to doi:
Removed doi.org prefix

**Mapped all genes and proteins to HGNC**
CD3E	CD3 epsilon subunit of T-cell receptor complex	HGNC:1674	
CD79A	CD79a molecule	HGNC:1698	
CLDN5	claudin 5	HGNC:2047	
DCN	decorin	HGNC:2705	
FOXJ1	forkhead box J1	HGNC:3816	
JCHAIN	joining chain of multimeric IgA and IgM	HGNC:5713	
KLRC1	killer cell lectin like receptor C1	HGNC:6374	
LYZ	lysozyme	HGNC:6740	
PAX8	paired box 8	HGNC:8622	
ACTA2	actin alpha 2, smooth muscle	HGNC:130	
TPSAB1	tryptase alpha/beta 1	HGNC:12019	
DES	desmin	HGNC:2770	
OVGP1	oviductal glycoprotein 1	HGNC:8524	
ENG	endoglin	HGNC:3349	CD105
ITGAX	integrin subunit alpha X	HGNC:6152	CD11c
CD163	CD163 molecule	HGNC:1631	
CD207	CD207 molecule	HGNC:17935	
CD44	CD44 molecule (Indian blood group)	HGNC:1681	
CD56	neural cell adhesion molecule 1	HGNC:7656	
FCGR1A	Fc gamma receptor Ia	HGNC:3613	CD64
NT5E	5'-nucleotidase ecto	HGNC:8021	CD73
THY1	Thy-1 cell surface antigen	HGNC:11801	CD90
EPCAM	epithelial cell adhesion molecule	HGNC:11529	
HLA-DRA	major histocompatibility complex, class II, DR alpha	HGNC:4947	HLA-DR
ITGA6	integrin subunit alpha 6	HGNC:6142	
VEGF	vascular endothelial growth factor A	HGNC:12680

**Reply by anitacaron** [reply]
Thank you so much for this effort, Ellen!

**Comment by emquardokus** [comment]
Fallopian_Tube_v1.2_DRAFT
https://docs.google.com/spreadsheets/d/16DHu7R9MC2B_fP7uRiDYcDIUvFs6SwHmZWgwpzuJwfI/edit#gid=991519552

**Blank ontology ID missing parent**

ADDED mesenchymal cell	CL:0008019   as CT/1  In row 14, 30, 48, 62, 85 the term fibroblast-mesenchymal cell without ontology ID has no parent that is from the CL ontology.

**Comment by emquardokus** [comment]
**Fallopian_Tube_v1.4_DRAFT**
https://docs.google.com/spreadsheets/d/1yCfVQmd3-R8wEN6qolpBT3w7hN2ug6pHyoxnM4i-sws/edit?gid=991519552#gid=991519552

**Different labels**
1. Fixed 19 instances, recently updated in Uberon. The term UBERON:8410010 has a different name/label in the source ontology in the following 32 rows 83, 84, 84, 85, 86, 86, 87, 87, 88, 88, 89, 89, 90, 90, 91, 91, 92, 92, 93, 93, 94, 94, 95, 95, 96, 96, 97, 98, 98, 116, 127, 128. The name/label in the ASCT+B table is fimbria of uterine tube and the one in the ontology is fimbria of fallopian tube. For reference, the given name/label by SMEs is fimbria of Fallopian tube. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2. Fixed 24 instances, recently updated in Uberon. The term UBERON:0012648 has a different name/label in the source ontology in the following 38 rows 44, 45, 45, 46, 46, 47, 47, 48, 48, 49, 49, 50, 50, 51, 51, 52, 52, 53, 53, 54, 54, 55, 55, 56, 56, 57, 57, 58, 59, 59, 81, 82, 112, 113, 114, 115, 125, 126. The name/label in the ASCT+B table is ampulla of uterine tube and the one in the ontology is ampulla of fallopian tube. For reference, the given name/label by SMEs is ampulla of Fallopian tube. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Fallopian_Tube_v1.4_DRAFT
https://docs.google.com/spreadsheets/d/1yCfVQmd3-R8wEN6qolpBT3w7hN2ug6pHyoxnM4i-sws/edit?gid=991519552#gid=991519552

**Different labels**

1. fixed 3 additional instances fimbria of fallopian tube	UBERON:8410010 The term UBERON:8410010 has a different name/label in the source ontology in the following 3 rows 116, 127, 128. The name/label in the ASCT+B table is fimbria of uterine tube and the one in the ontology is fimbria of fallopian tube. For reference, the given name/label by SMEs is fimbria of Fallopian tube. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
